### ISSUE TITLE
Implement crowd control

### ISSUE THREAD

**Issue by FoxxMD** [FoxxMD/context-mod#138]
https://www.reddit.com/dev/api/#POST_api_update_crowd_control_level

Implement updating crowd control on a submission as an Action

**Comment by Justusewhatevernameisstillavailable** [comment]
Looking forward to it!

Question, this does not include the toggle “Hold Crowd Controlled comments for review” right?

**Comment by FoxxMD** [comment]
It might...[this endpoint](https://www.reddit.com/dev/api/#POST_api_site_admin) contains these editable settings:

* crowd_control_chat_level
* crowd_control_filter
* crowd_control_level
* crowd_control_mode
* crowd_control_post_level

But, as usual, there is no description of what they actually do. I'll have to test it in my sandbox subreddit to see what they affect.

**Comment by Justusewhatevernameisstillavailable** [comment]
Yep, I meant for an individual post, sorry for not specifying that.

crowd_control_chat_level an integer between 0 and 3
crowd_control_level an integer between 0 and 3
crowd_control_post_level an integer between 0 and 3

these three probably match chat post, comment and post set to off, lenient, moderate or strict
